coverage/src/BamView-WebComponent.js

```diff
@@ -256,11 +265,11 @@ class BamViewChart extends HTMLElement {
             return;
         }
 
-        // Check if only the chromosome is provided and start/end are NaN
-        if (isNaN(start) && isNaN(end)) {
+        // Check if only the chromosome is provided and start and end inputs are empty
+        if (startInput === "" && endInput === "") {
```

Comment:
You want to check `parsedStart` and `parsedEnd` here. It's more idiomatic, and a undefined check should be faster than a string comparison.